.github/workflows/_build_test.yml

```diff
@@ -30,7 +51,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - run: ./build.sh ${{ matrix.component }}
+      # - uses: ./.github/actions/pull-ghcr
+      #   with:
+      #     github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - run: ./nx build ${{ matrix.component }}
```

Comment:
Because our tests have to run from a Docker container (to access the Docker network for the services under test), we can't really parallelise effectively here and using multiple GH Action machines is our friend for now.

Reply:
I thought about the build step, not about the test step. I thought the build could use `run-many` before uploading the built images to GHCR instead of building each image on a separate machine.

Reply:
I think this is not resolved. I'd really love to see nx in action here, as this is one of the core elements of our pipeline.

Reply:
This step will go back to a nx command once we sort out the publishing workflow.

Reply:
Yes, matrix is ~20% faster than single machine parallelization.